osmaxx/conversion/converters/converter_gis/sql_tables.py

```diff
@@ -0,0 +1,36 @@
+VIEW_NAMES = [
```

Comment:
the answer to the why: the original idea was to store the actual table names that should be included, hence the name. Will rename this to `OUTPUT_LAYER_NAMES` and the module to `layers`.